### PR TITLE
Is 73

### PR DESCRIPTION
**Pull request by engin64akarsu**

### CODE REVIEW

**Comment by engin64akarsu** [comment]
@seylab

File: src/test/java/com/ismakinesi/stepDefinitions/HomeStepDef.java [hunk 1]
```diff
@@ -250,4 +249,17 @@ public void kullaniciMarkaOlarakSkyjackRadioButtonunuSecer() {
         homePage.skyjack.click();
         BrowserUtilities.waitFor(4);
     }
+
+    @When("Teleskopik Yukleyici e tiklar")
+    public void teleskopikYükleyiciETiklar() {
```
Comment (seylab) [inline src/test/java/com/ismakinesi/stepDefinitions/HomeStepDef.java:254]: image
254. satır teleskopikYükleyiciETiklar()  **ü**

File: src/test/resources/features/002_HomePage/TümFiltrelerB.feature [hunk 2]
```diff
@@ -44,46 +44,66 @@ Feature: Tüm Filitreler
...
-      Given Kullanici ismakinesi.com adresine gider
-      When Yükleyiciler e tiklar
-      And Tum Filtreler e tiklar
-      And Kullanici kategori olarak Paletli Yükleyici Radio buttonunu secer
-      And Kullanici Marka olarak Caterpillar Radio buttonunu secer.
-      Then secilen ilanin filitrelendigini dogrular.
+  @IS-68
+  Scenario: TC:IS-68 Kullanici "Yükleyiciler" ilanlarini marka secerek secebilmelidir
+    Given Kullanici ismakinesi.com adresine gider
+    When Yükleyiciler e tiklar
```
Comment (seylab) [inline src/test/resources/features/002_HomePage/TümFiltrelerB.feature:50]: image

File: src/test/resources/features/002_HomePage/TümFiltrelerB.feature [hunk 3]
```diff
@@ -44,46 +44,66 @@ Feature: Tüm Filitreler
...
+    And Tum Filtreler e tiklar
+    And Kullanici kategori olarak Paletli Yükleyici Radio buttonunu secer
```
Comment (seylab) [inline src/test/resources/features/002_HomePage/TümFiltrelerB.feature:52]: image

File: src/test/resources/features/002_HomePage/TümFiltrelerB.feature [hunk 4]
```diff
@@ -44,46 +44,66 @@ Feature: Tüm Filitreler
...
+    And Kullanici Marka olarak Caterpillar Radio buttonunu secer.
+    Then secilen ilanin filitrelendigini dogrular.
+
+  @IS-69
+  Scenario: TC:IS-69 Kullanici "Yükleyiciler" ilanlarini marka secerek secebilmelidir
+    Given Kullanici ismakinesi.com adresine gider
+    When Yükleyici e tiklar
```
Comment (seylab) [inline src/test/resources/features/002_HomePage/TümFiltrelerB.feature:59]: image

File: src/test/resources/features/002_HomePage/TümFiltrelerB.feature [hunk 5]
```diff
@@ -44,46 +44,66 @@ Feature: Tüm Filitreler
...
+    And Tum Filtreler e tiklar
+    And Kullanici kategori olarak Lastik Tekerlekli Yükleyici Radio buttonunu secer
```
Comment (seylab) [inline src/test/resources/features/002_HomePage/TümFiltrelerB.feature:61]: image

File: src/test/resources/features/002_HomePage/TümFiltrelerB.feature [hunk 6]
```diff
@@ -44,46 +44,66 @@ Feature: Tüm Filitreler
...
-        And Tum Filtreler e tiklar
-        And Kullanici kategori olarak Lastik Tekerlekli Yükleyici Radio buttonunu secer
-        And Kullanici Marka olarak New Holland Radio buttonunu secer.
-        And Sinirli Servis Garantisi secer
-        And Uretim yili araligi  girer
-        And calisma saati araligi girer
-        And fiyat araligi girer
-        Then secilen ilanin filitrelendigini dogrular.
+  @IS-70
+  Scenario: TC:IS 70 Kullanici "Personel Yükselticiler" ilanlarini marka secerek görebilmelidir.
```
Comment (seylab) [inline src/test/resources/features/002_HomePage/TümFiltrelerB.feature:70]: image

File: src/test/resources/features/002_HomePage/TümFiltrelerB.feature [hunk 7]
```diff
@@ -44,46 +44,66 @@ Feature: Tüm Filitreler
...
+    Given Kullanici ismakinesi.com adresine gider
+    When Personel Yukselticiler e tiklar
+    And Tum Filtreler e tiklar
+    And Kullanici kategori olarak Akülü Makasli Platform Radio buttonunu secer
```
Comment (seylab) [inline src/test/resources/features/002_HomePage/TümFiltrelerB.feature:74]: image

File: src/test/resources/features/002_HomePage/TümFiltrelerB.feature [hunk 8]
```diff
@@ -44,46 +44,66 @@ Feature: Tüm Filitreler
...
-        @IS-70
-        Scenario: TC:IS 70 Kullanici "Personel Yükselticiler" ilanlarini marka secerek görebilmelidir.
-          Given Kullanici ismakinesi.com adresine gider
-          When Personel Yükselticiler e tiklar
-          And Tum Filtreler e tiklar
-          And Kullanici kategori olarak Akülü Makasli Platform Radio buttonunu secer
-          And Kullanici Marka olarak Genie Radio buttonunu secer.
-          Then secilen ilanin filitrelendigini dogrular.
+  @IS-71
+  Scenario: TC:IS 71 Kullanici "Personel Yükselticiler" ilanlarini marka secerek görebilmelidir.
```
Comment (seylab) [inline src/test/resources/features/002_HomePage/TümFiltrelerB.feature:79]: image

File: src/test/resources/features/002_HomePage/TümFiltrelerB.feature [hunk 9]
```diff
@@ -44,46 +44,66 @@ Feature: Tüm Filitreler
...
-            And Kullanici kategori olarak Dizel akulu platform Radio buttonunu secer
-            And Kullanici Marka olarak Skyjack Radio buttonunu secer.
-            And  Sinirli Servis Garantisi secer
-            And Uretim yili araligi  girer
-            And calisma saati araligi girer
-            And fiyat araligi girer
-            Then secilen ilanin filitrelendigini dogrular.
\ No newline at end of file
+  @IS-72
+  Scenario: TC:IS-72 Kullanici "Teleskopik Yükleyici" ilanlarini marka secerek görebilmelidir
```
Comment (seylab) [inline src/test/resources/features/002_HomePage/TümFiltrelerB.feature:92]: image

**Review by seylab — CHANGES_REQUESTED** [review]
@engin64akarsu

File: src/test/java/com/ismakinesi/stepDefinitions/HomeStepDef.java (hunk 1 above)
Comment (seylab) [reply src/test/java/com/ismakinesi/stepDefinitions/HomeStepDef.java:254]: @engin64akarsu

File: src/test/resources/features/002_HomePage/TümFiltrelerB.feature (hunk 2 above)
Comment (seylab) [reply src/test/resources/features/002_HomePage/TümFiltrelerB.feature:50]: @engin64akarsu

**Review by seylab — APPROVED** [review]

**Pull request merged**